pallets/stake/src/lib.rs

```diff
@@ -14,7 +14,30 @@
 // You should have received a copy of the GNU General Public License
 // along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Minimal staking module with ordered validator selection
+//! # Stake
+//! Minimal staking pallet that implements ordered validator selection by total amount at stake
+//!
+//! ### Rules
+//! There is a new round every `BlocksPerRound` blocks.
+//!
+//! At the start of every round,
+//! * `IssuancePerRound` is distributed to validators for `BondDuration` rounds ago
+//! in proportion to the points they received in that round (for authoring blocks)
+//! * queued validator exits are executed
+//! * new set of validators is chosen from the candidates
```

Comment:
This (the set of validators) is computed at the start of the round and not at the end  in the previous round, is that correct ?

Reply:
It is computed right before the next round starts, but this is atomic; the next round does not start without this set being recomputed first.